CHANGELOG.md

```diff
@@ -19,36 +19,33 @@
 - Drop support for Python 3.7.0 + 3.7.1 ([#4584](https://github.com/dbt-labs/dbt-core/issues/4584), [#4585](https://github.com/dbt-labs/dbt-core/pull/4585), [#4643](https://github.com/dbt-labs/dbt-core/pull/4643))
 - Re-format codebase (except tests) using pre-commit hooks ([#3195](https://github.com/dbt-labs/dbt-core/issues/3195), [#4697](https://github.com/dbt-labs/dbt-core/pull/4697))
 - Add deps module README ([#4686](https://github.com/dbt-labs/dbt-core/pull/4686/))
+- Support click versions in the v7.x series ([#4681](https://github.com/dbt-labs/dbt-core/pull/4681))
```

Comment:
This change was under the `1.0.2` section which I didn't think was right (it hasn't been backported) so I moved it up to be under `1.1.0`. @jtcohen6 just checking you didn't want this in `1.0.2`

Reply:
Thanks for the tag. I would like this in `1.0.3`. Reasoning originally stated in https://github.com/dbt-labs/dbt-core/issues/4566#issuecomment-1024375260:

> there's good cause to make it in a v1.0 patch release, so that folks can upgrade to v1.0 for use with Airflow, and don't need to wait until v1.1. The change feels pretty low-stakes since click is only used by dbt init.

Open to disagreement, however